creator-cards/jan-de-vries.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Jan de Vries",
+  "type": "CREATOR",
+  "countryCode": "nl",
+  "slogan": "If it doesn't go the way it should, it should go the way it can.",
+  "tags": "Azure, .NET, C#, ASP.NET, DevOps",
+  "github": "https://github.com/jandev/",
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://jan-v.nl/"
+    },
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/@Jandev"
+    },
+    {
+      "name": "Mastodon",
+      "url": "https://mastodon.cloud/@Jandev"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/jandv/"
+    }
+  ],
+  "feeds": [
+    {
+      "type": "LinkTagRssType",
+      "url": "https://jan-v.nl/index.xml"
+    },
+    {
+      "type": "LinkTagRssType",
+      "url": "https://www.youtube.com/feeds/videos.xml?channel_id=UCLfd7HBMLYJ0Muou1ihl_DA"
```

Comment:
hey @Jandev great so see you!
Any chance you can give us a feed filtered for .NET (or Azure) content? 
Otherwise I'll add your feeds as personal, thats ok as well. 

We have the rule that all "non" personal feeds have to be purely about .NET and related content. 
Personal allows it  all but only appears on your profile.

Reply:
Sure, will make that change. Both are dotnet and Azure related.

Copied the current values from others' documents, which indeed didn't match te template.

Just stepped away from the computer, so will do that tomorrow.